pkg/collector/corechecks/network-devices/cisco-sdwan/report/metrics.go

```diff
@@ -88,6 +90,7 @@ func (ms *SDWanSender) SendInterfaceMetrics(interfaceStats []client.InterfaceSta
 			index, err := itf.Index()
 			if err == nil {
 				tags = append(tags, fmt.Sprintf("interface_index:%d", index))
+				tags = append(tags, fmt.Sprintf("dd.internal.resource:ndm_interface_user_tags:%s:%s:%d", ms.namespace, entry.VmanageSystemIP, index))
```

Comment:
Should this use deviceUserTagResourcePrefix?

Reply:
nop, it is a different prefix, but to make it more clear I added another const 7a88b4d1b4004c7658f75936bac46a05e979fe43